### ISSUE TITLE
Table 11401 CBG Statement Line - procedure SetApplyCVLedgerEntries made protected

### ISSUE THREAD

**Issue by alinat2022** [microsoft/ALAppExtensions#24323]
Hello.

In #19494, it was requested that procedure SetApplyCVLedgerEntries should have the accessibility level changed from local to protected.

In 22.3 version, this is still not implemented. Can you please have a look? Thank you.
